appengine/gophers/gophers-2/app.yaml

```diff
@@ -2,7 +2,15 @@ runtime: go
 api_version: go1
 
 handlers:
+// [START static_handlers]
+# If the path is empty, show static file index.html
+- url: /
+  static_files: index.html
+  upload: index.html
+  
+# Otherwise, find file in static-files directory
 - url: /static
-  static_dir: static
+  static_dir: static-files
```

Comment:
I think this was correct before?

Reply:
Thinking about changing the folder name to static-files so users know that the folder name doesn't have to be the same as the URL. WDYT?
Also, took out the template variable in gophers-2/main.go, so I can introduce templates all at once in next page of tutorial.

Reply:
Keeping it to "static" makes the most sense. They can play around with it if they need to, but keeping the directory the same as the eventual path is going to be what most people want.